internal/operands/metrics/reconcile.go

```diff
@@ -23,12 +23,16 @@ func (m *metrics) WatchClusterTypes() []runtime.Object {
 	return nil
 }
 
-func (m *metrics) Reconcile(request *common.Request) error {
-	return common.CreateOrUpdateResource(request,
+func (m *metrics) Reconcile(request *common.Request) ([]common.ResourceStatus, error) {
+	status, err := common.CreateOrUpdateResource(request,
```

Comment:
Not sure if it's in the scope of this PR, but maybe use `Reconcile()` with a `reconcilePrometheusRule()` function?
For alignment with the rest of the operands

Reply:
Done